Source/WebCore/css/CSSVariableReferenceValue.h

```diff
@@ -68,6 +68,9 @@ class CSSVariableReferenceValue : public CSSValue {
 
     template<typename CacheFunction> bool resolveAndCacheValue(Style::BuilderState&, CacheFunction&&) const;
 
+    // Say true conservatively.
+    bool customMayDependOnBaseURL() const { return true; }
```

Comment:
Same question here.

Reply:
On the other hand, I think this is not necessary to prohibit it. I'll drop this.